README.md

```diff
@@ -11,7 +11,7 @@ We would like to have guides for the following tasks.
 * Full Syntactic Analysis
 * [Named Entity Recognition](named-entity-recognition/)
 * Machine Translation
-
+* [General Knowledge](general-knowledge/)
```

Comment:
I think that in this `README.md` it'd be better to have a new section/bullet list for "General Knowledge", where "Data Collection" is a separate tab. I think the directory structure is OK.